fts5_vs_orama_search_demo.html

```
@@ -0,0 +1,482 @@
+.vecval{min-width:44px;text-align:right;color:var(--color-text-secondary);font-family:var(--font-mono)}
+.empty{padding:32px 0;text-align:center;color:var(--color-text-secondary);font-size:12px}
+.idxrow{display:flex;justify-content:space-between;font-size:11px;padding:2.5px 0;border-bottom:0.5px solid var(--color-border-tertiary);font-family:var(--font-mono)}
+.idxrow:last-child{border-bottom:none}
+.idxterm{color:var(--color-text-info)}.idxmeta{color:var(--color-text-secondary)}
+.info-tip{font-size:12px;color:var(--color-text-secondary);background:var(--color-background-secondary);border:0.5px solid var(--color-border-tertiary);border-radius:var(--border-radius-md);padding:9px 12px;margin-bottom:12px;line-height:1.6}
+.info-tip b{color:var(--color-text-primary);font-weight:500}
+</style>
+
+<div style="padding:1rem 0">
+<div class="tabs">
+  <button class="tab-btn active" id="btn-fts5" onclick="showTab('fts5')">⚡ SQLite FTS5</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)